test/build/resolve-js/src/a/a.js

```diff
@@ -0,0 +1 @@
+import '/index.js'; // import root
```

Comment:
@FredKSchott added a test for this anyway, but this was working as expected! No problems here.